### ISSUE TITLE
ApiResponseSchema.responseDescription ignored

### ISSUE THREAD

**Issue by Azquelt** [smallrye/smallrye-open-api#618]
When using the `@APIResponseSchema` annotation with a `responseDescription`, the description is ignored and replaced by the description of the response code.

Example:
```
    @APIResponseSchema(responseCode = "200",
                       responseDescription = "My Custom Response",
                       value = Properties.class)
```
Result in the OpenAPI document:
```
      responses:
        "200":
          description: OK
          content:
            application/json:
              schema:
                type: object
```
Expected:
```
      responses:
        "200":
          description: My Custom Response
          content:
            application/json:
              schema:
                type: object
```

I had a look through the code and couldn't see any references to `ResponseConstant.PROP_RESPONSE_DESCRIPTION` or any tests which use `ApiResponseSchema.responseDescription`

**Comment by MikeEdgar** [comment]
@Azquelt - thanks for finding this. The fix will be in the 2.1.0 release. Are you testing with 2.1.0-RC1?

**Comment by Azquelt** [comment]
No, I think we're currently using an older snapshot and need to update to 2.1.0-RC1. I did have to check that this hadn't already been fixed before raising the issue.
